### PR TITLE
feat(route): add route groups along with its pages and tests

### PR DESCRIPTION
**Pull request by ishaan000**
- Add access folder  as a [route group](https://nextjs.org/docs/app/building-your-application/routing/route-groups) for cleaner URL.
- Add folders inside access with respective page.tsx files.
- Create tests for access routes  to check their rendering.

### CODE REVIEW

File: tests/app/access/account/account.test.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Account from '../../../../src/app/(access)/account/page';
+import React from 'react';
+import { render } from '@testing-library/react';
+
+describe('Account Page', () => {
+  it('renders the Account component without crashing', () => {
```
Comment (hiyaryan) [inline tests/app/access/account/account.test.tsx:6]:
```suggestion
  it('renders the Account component', () => {
```

Comment (hiyaryan) [reply tests/app/access/account/account.test.tsx:6]: It's implied that if the component renders it didn't crash.

File: tests/app/access/forgot-password/forgot-password.test.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import ForgotPassword from '../../../../src/app/(access)/forgot-password/page';
+import React from 'react';
+import { render } from '@testing-library/react';
+
+describe('Forgot Password Page', () => {
+  it('renders the ForgotPassword component without crashing', () => {
```
Comment (hiyaryan) [inline tests/app/access/forgot-password/forgot-password.test.tsx:6]:
```suggestion
  it('renders the ForgotPassword component', () => {
```

File: tests/app/access/login/login.test.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import Login from '../../../../src/app/(access)/login/page';
+import React from 'react';
+import { render } from '@testing-library/react';
+
+describe('Login Page', () => {
+  it('renders the Login component without crashing', () => {
```
Comment (hiyaryan) [inline tests/app/access/login/login.test.tsx:6]:
```suggestion
  it('renders the Login component', () => {
```

File: tests/app/access/register/register.test.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Register from '../../../../src/app/(access)/register/page';
+import { render } from '@testing-library/react';
+
+describe('Register Page', () => {
+  it('renders the Register component without crashing', () => {
```
Comment (hiyaryan) [inline tests/app/access/register/register.test.tsx:6]:
```suggestion
  it('renders the Register component', () => {
```

File: tests/app/access/reset-password/reset-password.test.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ResetPassword from '../../../../src/app/(access)/reset-password/page';
+import { render } from '@testing-library/react';
+
+describe('Reset Password Page', () => {
+  it('renders the ResetPassword component without crashing', () => {
```
Comment (hiyaryan) [inline tests/app/access/reset-password/reset-password.test.tsx:6]:
```suggestion
  it('renders the ResetPassword component', () => {
```

**Review by hiyaryan — CHANGES_REQUESTED** [review]
Great work Ishaan @ishaan000! Looks like a great start. I'm requesting a few changes to the test names for brevity. Please take a look when you get the chance 😁

File: tests/app/access/reset-password/reset-password.test.tsx (hunk 5 above)
Comment (ishaan000) [reply tests/app/access/reset-password/reset-password.test.tsx]: Done! 😌

**Review by hiyaryan — APPROVED** [review]
Looks good! Thank you for updating the test names and great work laying the foundation @ishaan000

**Pull request merged**